src/rageshake/submit-rageshake.ts

```diff
@@ -87,10 +87,12 @@ async function collectBugReport(opts: IOpts = {}, gzipLogs = true): Promise<Form
         if (cryptoApi) {
             body.append("crypto_version", cryptoApi.getVersion());
 
-            const keys = [`ed25519:${client.getDeviceEd25519Key()}`];
-            if (client.getDeviceCurve25519Key) {
-                keys.push(`curve25519:${client.getDeviceCurve25519Key()}`);
-            }
+            const ownDeviceKeys = await cryptoApi.getOwnDeviceKeys();
+
+            const keys = [];
+            keys.push(`curve25519:${ownDeviceKeys.curve25519}`);
+            keys.push(`ed25519:${ownDeviceKeys.ed25519}`);
```

Comment:
minor suggestion:

```suggestion
            const ownDeviceKeys = await cryptoApi.getOwnDeviceKeys();
            const keys = [
                `curve25519:${ownDeviceKeys.curve25519}`,
                `ed25519:${ownDeviceKeys.ed25519}`,
            ];
```

simpler and clearer IMHO.